core/modules/system/system.admin.inc

```diff
@@ -1604,25 +1582,125 @@ function system_run_cron_submit($form, &$form_state) {
 }
 
 /**
- * Form builder; Configure error reporting settings.
+ * Backwards compatible wrapper function.
+ * @todo Remove in Backdrop 2.x.
+ */
+function system_logging_settings($form, &$form_state) {
+  watchdog_deprecated_function('system', __FUNCTION__);
+  return system_error_settings($form, $form_state);
+}
+
+/**
+ * Form builder; Configure error handling settings.
  *
  * @ingroup forms
  * @see system_logging_settings_validate()
  */
-function system_logging_settings($form, &$form_state) {
+function system_error_settings($form, &$form_state) {
+  $config = config('system.core');
   $form['#config'] = 'system.core';
-  $form['error_level'] = array(
+
+  $form['error_display'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('PHP errors'),
+  );
+  $form['error_display']['error_level'] = array(
     '#type' => 'radios',
     '#title' => t('Error messages to display'),
-    '#default_value' => config_get('system.core', 'error_level'),
+    '#default_value' => $config->get('error_level'),
     '#options' => array(
       ERROR_REPORTING_HIDE => t('None'),
       ERROR_REPORTING_DISPLAY_SOME => t('Errors and warnings'),
       ERROR_REPORTING_DISPLAY_ALL => t('All messages'),
     ),
-    '#description' => t('It is recommended that sites running on production environments do not display any errors.'),
+    '#description' => t('It is recommended that sites running on production environments do not display any PHP errors.'),
+  );
+
+  $form['error_pages'] = array(
```

Comment:
You missed specifying $field_prefix, so it's not shown before the 403/404 autocomplete fields:

```suggestion
  $field_prefix = url(NULL, array('absolute' => TRUE)) . (config_get('system.core', 'clean_url') ? '' : '?q=');

  $form['error_pages'] = array(
```